extensions/amp-story/1.0/amp-story-open-page-attachment.js

```diff
@@ -37,7 +37,7 @@ const CtaAccentElement = {
  * @param {!Element} element
  * @return {!Element}
  */
-export const buildOpenDefaultAttachmentElement = (element) =>
+export const buildOldAttachmentElement = (element) =>
```

Comment:
I'm not sure 'Old' is the right name for this. Technically the old UI is still in use in the case of inline attachments that have no images.

Reply:
`buildOpenInlineAttachmentElement` would be the codepath for with and without images with the work in this PR.

The benefit of this is that they can share structure, style and animations.